e2e/__tests__/global_teardown.test.js

```diff
@@ -49,3 +49,19 @@ test('jest throws an error when globalTeardown does not export a function', () =
     `TypeError: globalTeardown file must export a function at ${teardownPath}`,
   );
 });
+
+test('globalTeardown function gets jest config object as a parameter', () => {
+  const teardownPath = path.resolve(
+    __dirname,
+    '../global-teardown/teardown-with-config.js',
+  );
+
+  const testPathPattern = 'custom_tests_dir/pass';
+
+  const result = runJest('global-teardown', [
+    `--globalTeardown=${teardownPath}`,
+    `--testPathPattern=${testPathPattern}`,
+  ]);
+
+  expect(result.stdout).toBe(testPathPattern);
```

Comment:
Maybe pass through `slash` to fix the os inconsistencies?

Reply:
Thanks for the advice, I realized that I can test it without giving a `path` to the `testPathPattern` so we won't have to deal with the inconsistency in the first place.